src/index.js

```diff
@@ -153,6 +169,8 @@ export default {
   activate() {
     require('atom-package-deps').install('linter-rubocop', true)
```

Comment:
As commented in the other issue putting this and the `loadDeps` into an idle callback is still useful even with the `activationHook` usage.